migrations/54-60/removed-backward-incompatibility.md

```diff
@@ -2,11 +2,18 @@
 sidebar_position: 3
 ---
 
-# Removed and backward incompatibility
+Removed and backward incompatibility
+====================================
 
 :::tip[Developer Note]
   Since this version of Joomla has not been released yet, this page can change anytime.
 :::
 
 All the deprecated features than have now been removed and any backward incompatibilities.
 There should be an explanation of how to mitigate the removals / changes.
+
+### CMS Input object switched to Framework Input object
+The CMS Input object `\Joomla\CMS\Input` has been deprecated since Joomla 4.3. The CMS core code
```

Comment:
```suggestion
### CMS Input object switched to Framework Input object

The CMS Input object `\Joomla\CMS\Input` has been deprecated since Joomla 4.3. The CMS core code
```